parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -4853,6 +4857,36 @@ SqlIfStmt IfStmt() :
     }
 }
 
+SqlCaseStmt CaseStmt() :
+{
+    SqlNode firstOperand = null;
+    SqlNode e = null;
+    final SqlNodeList conditionMultiStmtList = new SqlNodeList(getPos());
+    final SqlStatementList elseMultiStmtList = new SqlStatementList(getPos());
```

Comment:
nit: Line wrap

Reply:
This line ends at 79 character, which is not against the style we are using (wrap at 80 character).

Reply:
You're right - looks good.